adapters/saml/wildfly/wildfly-subsystem/src/main/java/org/keycloak/subsystem/adapter/saml/extension/KeycloakSubsystemDefinition.java

```diff
@@ -31,7 +30,7 @@
 public class KeycloakSubsystemDefinition extends SimpleResourceDefinition {
 
     static final KeycloakSubsystemDefinition INSTANCE = new KeycloakSubsystemDefinition();
-    static final ModuleIdentifier KEYCLOAK_JBOSS_CORE_ADAPTER = ModuleIdentifier.create("org.keycloak.keycloak-jboss-adapter-core");
+    static final String KEYCLOAK_JBOSS_CORE_ADAPTER = "org.keycloak.keycloak-jboss-adapter-core";
```

Comment:
Use string defined in KeycloakDependencyProcessor (not to duplicate module identifiers).